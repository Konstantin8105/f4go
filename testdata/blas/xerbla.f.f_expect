
 SUBROUTINE XERBLA ( SRNAME , INFO )
 CHARACTER * ( * ) SRNAME
 INTEGER INFO
 INTRINSIC LEN_TRIM
 WRITE ( * , FMT = 9999 ) SRNAME ( 1 : LEN_TRIM ( SRNAME ) ) , INFO
 STOP
 9999 FORMAT ( ' ** On entry to ' , A , ' parameter number ' , I2 , ' had ' ,
 'an illegal value' )
 END
