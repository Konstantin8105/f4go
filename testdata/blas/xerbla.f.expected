COMMENT             	*> \brief \b XERBLA
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*  =========== DOCUMENTATION ===========
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	* Online html documentation available at
NEW_LINE            	

COMMENT             	*            http://www.netlib.org/lapack/explore-html/
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*  Definition:
NEW_LINE            	

COMMENT             	*  ===========
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*       SUBROUTINE XERBLA( SRNAME, INFO )
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*       .. Scalar Arguments ..
NEW_LINE            	

COMMENT             	*       CHARACTER*(*)      SRNAME
NEW_LINE            	

COMMENT             	*       INTEGER            INFO
NEW_LINE            	

COMMENT             	*       ..
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*> \par Purpose:
NEW_LINE            	

COMMENT             	*  =============
NEW_LINE            	

COMMENT             	*>
NEW_LINE            	

COMMENT             	*> \verbatim
NEW_LINE            	

COMMENT             	*>
NEW_LINE            	

COMMENT             	*> XERBLA  is an error handler for the LAPACK routines.
NEW_LINE            	

COMMENT             	*> It is called by an LAPACK routine if an input parameter has an
NEW_LINE            	

COMMENT             	*> invalid value.  A message is printed and execution stops.
NEW_LINE            	

COMMENT             	*>
NEW_LINE            	

COMMENT             	*> Installers may consider modifying the STOP statement in order to
NEW_LINE            	

COMMENT             	*> call system-specific exception-handling facilities.
NEW_LINE            	

COMMENT             	*> \endverbatim
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*  Arguments:
NEW_LINE            	

COMMENT             	*  ==========
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*> \param[in] SRNAME
NEW_LINE            	

COMMENT             	*> \verbatim
NEW_LINE            	

COMMENT             	*>          SRNAME is CHARACTER*(*)
NEW_LINE            	

COMMENT             	*>          The name of the routine which called XERBLA.
NEW_LINE            	

COMMENT             	*> \endverbatim
NEW_LINE            	

COMMENT             	*>
NEW_LINE            	

COMMENT             	*> \param[in] INFO
NEW_LINE            	

COMMENT             	*> \verbatim
NEW_LINE            	

COMMENT             	*>          INFO is INTEGER
NEW_LINE            	

COMMENT             	*>          The position of the invalid parameter in the parameter list
NEW_LINE            	

COMMENT             	*>          of the calling routine.
NEW_LINE            	

COMMENT             	*> \endverbatim
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*  Authors:
NEW_LINE            	

COMMENT             	*  ========
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*> \author Univ. of Tennessee
NEW_LINE            	

COMMENT             	*> \author Univ. of California Berkeley
NEW_LINE            	

COMMENT             	*> \author Univ. of Colorado Denver
NEW_LINE            	

COMMENT             	*> \author NAG Ltd.
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*> \date December 2016
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*> \ingroup aux_blas
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*  =====================================================================
NEW_LINE            	

SUBROUTINE          	SUBROUTINE
IDENT               	XERBLA
(                   	(
IDENT               	SRNAME
,                   	,
IDENT               	INFO
)                   	)
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*  -- Reference BLAS level1 routine (version 3.7.0) --
NEW_LINE            	

COMMENT             	*  -- Reference BLAS is a software package provided by Univ. of Tennessee,    --
NEW_LINE            	

COMMENT             	*  -- Univ. of California Berkeley, Univ. of Colorado Denver and NAG Ltd..--
NEW_LINE            	

COMMENT             	*     December 2016
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*     .. Scalar Arguments ..
NEW_LINE            	

CHARACTER           	CHARACTER
*                   	*
(                   	(
*                   	*
)                   	)
IDENT               	SRNAME
NEW_LINE            	

INTEGER             	INTEGER
IDENT               	INFO
NEW_LINE            	

COMMENT             	*     ..
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	* =====================================================================
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*     .. Intrinsic Functions ..
NEW_LINE            	

IDENT               	INTRINSIC
IDENT               	LEN_TRIM
NEW_LINE            	

COMMENT             	*     ..
NEW_LINE            	

COMMENT             	*     .. Executable Statements ..
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

WRITE               	WRITE
(                   	(
*                   	*
,                   	,
IDENT               	FMT
=                   	=
INT                 	9999
)                   	)
IDENT               	SRNAME
(                   	(
INT                 	1
:                   	:
IDENT               	LEN_TRIM
(                   	(
IDENT               	SRNAME
)                   	)
)                   	)
,                   	,
IDENT               	INFO
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

IDENT               	STOP
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

INT                 	9999
IDENT               	FORMAT
(                   	(
STRING              	' ** On entry to '
,                   	,
IDENT               	A
,                   	,
STRING              	' parameter number '
,                   	,
IDENT               	I2
,                   	,
STRING              	' had '
,                   	,
NEW_LINE            	

STRING              	'an illegal value'
)                   	)
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*     End of XERBLA
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

END                 	END
NEW_LINE            	

